File: MyHeart.Web/ClientApp/models/user/UserDto.ts
export default class UserDto {
    id: number;
    firstName: string | null;
    lastName: string | null;
    email: string | null;
    userType: UserType | null;
    stripeId: string;
    userTypes: string | null;
}
export enum UserType {
    SuperAdmin = 0,
    Admin = 1,
    Doctor = 2,
    Patient = 3,
    DataManager = 4
}

File: MyHeart.DTO/User/UserDTO.cs
using MyHeart.DTO.MedicalReport;
using MyHeart.DTO.Questions;
using System;

namespace MyHeart.DTO.User
{
    public class UserDTO : BaseEntityDTO
    {
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public UType UserType { get; set; }
        public string StripeId { get; set; }
        public virtual QuestionListDTO Question { get; set; }
        public int SubscriptionPreferences { get; set; }
        public string UserTypes { get; set; }
        public string FullName  => $"{FirstName} {LastName}";
    }

    public enum UType {
        SuperAdmin,
        Admin,
        Doctor,
        Patient,
        DataManager
    }
  
}